src/extensions/default/bramble/main.js

```diff
@@ -120,6 +121,13 @@ define(function (require, exports, module) {
 
         // We're all done loading and can pass startup state info back to the host app.
         RemoteEvents.loaded();
+
+        // Show Tutorial Pane by default if `tutorial.html` exists at project load
```

Comment:
Let's move this code change into https://github.com/mozilla/brackets/blob/master/src/extensions/default/bramble/lib/RemoteEvents.js#L211-L246.  Ideally we can send the value in our initial `sendEvent` vs. needing to fire a second event just for this.